src/Microsoft.Diagnostics.Monitoring.WebApi/Operation/InProcessEgressOperation.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Diagnostics.Monitoring.WebApi
+{
+    internal sealed class InProcessEgressOperation : IEgressOperation
+    {
+        private readonly KeyValueLogScope _scope;
+
+        public EgressProcessInfo ProcessInfo { get; private set; }
+        public string EgressProviderName { get { return null; } }
+        public bool IsStoppable { get { return _operation?.IsStoppable ?? false; } }
```

Comment:
```suggestion
        public bool IsStoppable => _operation.IsStoppable;
```
I don't think you have to check for null.